b2share/modules/communities/helpers.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of EUDAT B2Share.
+# Copyright (C) 2016 CERN.
+#
+# B2Share is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# B2Share is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with B2Share; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Helper functions for B2Share Communities module."""
+
+from sqlalchemy import func
+
+from .api import Community
+from .errors import CommunityDoesNotExistError
+
+def get_community_by_name_or_id(community):
+    result = None
+    try:
+        result = Community.get(id=community)
+    except:
+        try:
+            #TODO: case insensitivity?
```

Comment:
I wouldn't try to make this too complex for now. This CLI will **only** be used by the admin. Most of the time he will probably use the ID.

Also the current `name` of the community will probably be matched to the `title` field in invenio-communities model. This field has a length of 255 characters. I doubt that the admin will use such a long value. Maybe later we can see if a prefix can be used when it is unique, but this is more a bonus feature if it is asked for.